### PR TITLE
Upgrade Glide from v3 to v4

### PR DESCRIPTION
**Pull request by rickymohan**
Temporary fix to upgrade Glide to v4

### CODE REVIEW

**Comment by kayako-service-user** [comment]
Can one of the admins verify this patch?

**Comment by alexandra-pana** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by rickymohan** [comment]
Thanks for the review. We have addressed the issues which had commented code and was breaking the the attachment preview functionality.

File: ProjectSource/app-library/src/main/java/com/kayako/sdk/android/k5/common/adapter/messengerlist/helper/AttachmentHelper.java [hunk 1]
```diff
@@ -134,7 +132,7 @@ private static void setUpAttachmentCaption(String caption, TextView captionTextV
     }
 
     private static void configureAttachmentImage(ImageView imageView, String imageUrl) {
-        ImageUtils.loadUrlAsAttachmentImage(Kayako.getApplicationContext(), imageView, imageUrl, true, true, null, null);
+      //  ImageUtils.loadUrlAsAttachmentImage(Kayako.getApplicationContext(), imageView, imageUrl, true, true, null, null);
```
[the remark this answers is not in the document]
Comment (rickymohan) [reply ProjectSource/app-library/src/main/java/com/kayako/sdk/android/k5/common/adapter/messengerlist/helper/AttachmentHelper.java]: This has been addressed in the latest commit

**Pull request closed without merging**